src/components/Composer/Post/Update/index.tsx

```diff
@@ -191,36 +203,62 @@ const NewUpdate: FC = () => {
     if (!currentProfile) {
       return toast.error(SIGN_WALLET);
     }
+
+    if (isAudioPost) {
+      setPostContentError('');
+      const AudioPublicationSchema = z.object({
+        title: z.string().trim().min(1, { message: 'Invalid audio title' }),
+        author: z.string().trim().min(1, { message: 'Invalid author name' }),
+        cover: z.string().trim().min(1, { message: 'Invalid cover image' })
+      });
```

Comment:
**suggestion:** can we keep it somewhere and reuse it in both places 🙏🏼

Reply:
Done